devlogs/raisa/daily-logs/2025-03-16.md

```
@@ -0,0 +1,320 @@
+In this code, you can add any shape without making any change to the AreaCalculator class.
+
+# Liskov Substitution
+
+“**Derived or child classes must be substitutable for their base or parent classes**“
+
+Breaking it down simply:
+
+- If **class B** is a subclass of **class A**, then we should be able to replace **A** with **B** *without breaking the functionality*.
+- The **child class** should **extend** the parent class **without changing its expected behavior**.
+
+**An Example of Violation**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)